src/components/settings-menu/DevSection.js

```diff
@@ -33,6 +37,7 @@ const DevSection = () => {
       );
       logger.log('connected to ganache', ready);
     } catch (e) {
+      await web3SetHttpProvider(networkTypes.mainnet);
```

Comment:
what's the logic behid this? if ganache fails to initialize the app will connect to mainnet and spend real ETH, which is a problem

Reply:
this prevents a false positive, without this line the ganache toast will appear even if there is an error. Lmk if theres a better way!

Reply:
Oh, I see. Your test checks that the toast saying Ganache is there. If that's the case then it's ok.